game-backend/src/MainController.ts

```diff
@@ -1,18 +1,63 @@
 import WebSocket from 'ws';
 import hyperid from 'hyperid';
 import { GameObject } from '../../darwin-types/GameObject';
+import { UserContext } from '../../darwin-types/UserContext';
+import { ServerStore, createStore } from './ServerStore';
 
-const hyperIdInstance = hyperid();
-
+/**
+ * Main controller, which handles new connections and stores match data and other server data.
+ */
 export default class MainController {
-  static newConnection(ws: WebSocket): void {
+  private hyperIdInstance: hyperid.Instance = hyperid();
+
+  private store: ServerStore = createStore();
```

Comment:
Shall we add the types explicitly even if it's not required?

Reply:
Yes please. IMO types should be there only if not initialized. What do you guys think?

Reply:
I dont really see the point in adding types which the IDE already can infer.
What is the point in doing so?